### ISSUE TITLE
Unable to communicate with Python application running within a virtualenv

### ISSUE THREAD

**Issue by swinton** [awslabs/amazon-kinesis-client-python#6]
Using the [supplied samples](https://github.com/awslabs/amazon-kinesis-client-python/tree/master/samples), I am writing a Kinesis application, running my Python code within a [`virtualenv`](https://virtualenv.pypa.io/). However, it seems the Java application is unable to communicate with my application when running within a virtualenv.

Is there a configuration available that would allow this?

**Comment by swinton** [comment]
Closing, didn't realize the executables were installed under `/usr/local/bin`.

**Comment by knorwood** [comment]
Hi @swinton,

If you could elaborate a bit more on what your issue was and what you did to solve it, I might be able to improve the getting started documentation some more.

Thanks,
Kurt

**Comment by swinton** [comment]
Sure...

I was editing `sample_kclpy_app.py` under the `samples` dir, couldn't figure out why my changes weren't getting picked up.

I assumed (incorrectly) it was because I was running under a virtualenv, and the Java process couldn't find the correct python executable for this env.

I started over, using a VM so I could use the system-wide python environment, but when I _still_ had the same issue, I finally discovered that the Java process was calling out to `/usr/local/bin/sample_kclpy_app.py`. To address this, I edited `sample.properties` to point to my local copy.

My rough notes describing the steps I took to get it running under a VM are [here](https://gist.github.com/swinton/cc70729441a7b54c62ec).
